docs/kafkacat-usage.rst

```diff
@@ -0,0 +1,137 @@
+kafkacat
```

Comment:
one more thing, please add an RST anchor as the first line of the page: `.. _kafkacat-usage:`. this allows us to link to it from other pages using just this ID, not a hard-coded link (e.g. `:ref:`kafkcat-usage``).